opentelekomcloud/acceptance/iam/data_source_opentelekomcloud_identity_project_v3_test.go

```diff
@@ -43,6 +43,22 @@ func TestAccOpenStackIdentityV3ProjectDataSource_basic(t *testing.T) {
 	})
 }
 
+func TestAccOpenStackIdentityV3ProjectDataSource_empty(t *testing.T) {
+	var emptyProjectName, emptyProjectDescription string
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			common.TestAccPreCheck(t)
+		},
+		ProviderFactories: common.TestAccProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccOpenStackIdentityProjectV3DataSource_project(emptyProjectName, emptyProjectDescription),
```

Comment:
Please add a check for ID to be set, as it's done in `TestLBFlavorV3_basic`

Reply:
okey